packages/core/src/billing/billing-address-action-creator.ts

```diff
@@ -33,6 +33,10 @@ export default class BillingAddressActionCreator {
         return (store) => {
             const state = store.getState();
             const checkout = state.checkout.getCheckout();
+            const isBillingFixExperimentEnabled =
+                state.config.getConfig()?.storeConfig.checkoutSettings.features[
+                    'CHECKOUT-8392.fix_billing_creation_in_checkout'
+                ] ?? false;
```

Comment:
We might need to return `true` as fallback in order to safely remove an experiment in future.
Otherwise, removing the experiment will make the flow revert back to the time before the experiment rollout.

Reply:
I did think about it, but wasn't sure. Convinced now, let me update it.